packages/psss-server/src/routes/reports/ConfirmWeeklyReportRoute.ts

```diff
@@ -128,35 +126,30 @@ export class ConfirmWeeklyReportRoute extends Route {
       // archive the existing alert triggered in the selected week
       if (currentWeekSyndromeAlert && result[`${syndromeCode} Threshold Crossed`] === false) {
         await this.archiveAlert(currentWeekSyndromeAlert.id, countryCode, week);
+        response.alertsArchived = true;
       }
     }
 
     return response;
   }
 
   async createAlert(countryCode: string, week: string, syndromeCode: string) {
-    return this.meditrakConnection?.createSurveyResponse(ALERT_SURVEY, countryCode, week, {
-      PSSS_Alert_Syndrome: syndromeCode,
-      PSSS_Alert_Archived: 'No',
-    });
+    return this.meditrakConnection?.createSurveyResponse(ALERT_SURVEY, countryCode, week, [
+      { code: 'PSSS_Alert_Syndrome', type: 'Radio', value: syndromeCode },
```

Comment:
Good question @kael89! Yes, I think you're probably right - I have a vague memory that in the early days of meditrak we were changing questions all the time before things stabilised a bit, and wanted to know what type of answer it was even if the question type had changed. I can't be sure though as it was years ago and my memory is hazy - it may have been a simple denormalisation for a performance or code simplicity benefit. Given the way we use the data in that table now, it would be bad for lots of reasons if the answer type was different to the question type, so I totally agree that we should be able to remove this column.